src/omega_edit/utils.ts

```diff
@@ -94,7 +94,7 @@ export function logicalDisplay(
   bytes: ArrayBuffer,
   bytesPerRow: number
 ): string {
-  const undefinedCharStandIn = '�'
+  const undefinedCharStandIn = '░'
```

Comment:
Is there a way to use an ascii code that when converted to a string it will give this value? Just thinking if we used a ascii code or something then convert it to a string if it produces the same result it would look cleaner that having this weird symbol in there.

Reply:
@stricklandrbls Can we instead change this variable to be `const undefinedCharCodeStandIn = 9617`? Then when you go to use it do `String.fromCharCode(undefinedCharCodeStandIn)`? I was able to get this char code value by doing `console.log('░'.charCodeAt(0))` in a nodejs shell

Reply:
Sure thing